src/App.tsx

```diff
@@ -42,9 +41,35 @@ import {
 } from './components/DirectMessage';
 import { useAuth, AuthProvider } from './Auth';
 
-firebase.initializeApp(firebaseConfig);
+const initFirebase = async () => {
+  if (process.env.NODE_ENV === 'production') {
+    return fetch('/__/firebase/init.json').then(async (response) => {
```

Comment:
This is usually handled using environment variables which are set in CI and loaded from a .env file locally, but this method works as well!

Reply:
Yep, it slows a bit the initial start but allows easier setup, we could optimize later.